src/GridFS/WritableStream.php

```diff
@@ -262,6 +262,9 @@ private function abort()
         $this->isClosed = true;
     }
 
+    /**
+     * @return mixed
+     */
```

Comment:
In this case, I expect a doc block is required since there is no return type hint equivalent until [PHP 8.0](https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.mixed), which is also where union types are introduced.

Reply:
That is correct 👍